test/validate.test.js

```diff
@@ -1,7 +1,7 @@
 import assert from 'assert';
 import { Recurly } from '../lib/recurly';
 
-describe('Recurly.validate', function () {
+describe.only('Recurly.validate', function () {
```

Comment:
Remove the `.only`.

Reply:
but it runs faster this way tho :trollface: